src/probnum/diffeq/_perturbsolve_ivp.py

```diff
@@ -155,22 +196,50 @@ def perturbsolve_ivp(
     References
     ----------
     .. [1] Abdulle, A. and Garegnani, G..
-        Random time step probabilistic methods for uncertainty quantification in chaotic and geometric numerical integration.
+        Random time step probabilistic methods for uncertainty quantification
+        in chaotic and geometric numerical integration.
         Statistics and Computing. 2020.
     .. [2] J. R. Dormand, P. J. Prince..
         A family of embedded Runge-Kutta formulae.
-        Journal of Computational and Applied Mathematics, Vol. 6, No. 1, pp. 19-26, 1980.
+        Journal of Computational and Applied Mathematics,
+        Vol. 6, No. 1, pp. 19-26, 1980.
     .. [3] L. W. Shampine.
         Some Practical Runge-Kutta Formulas.
         Mathematics of Computation, Vol. 46, No. 173, pp. 135-150, 1986.
     .. [4] P. Bogacki, L.F. Shampine.
         A 3(2) Pair of Runge-Kutta Formulas.
         Appl. Math. Lett. Vol. 2, No. 4. pp. 321-325, 1989.
-    """
+    """  # pylint: disable="line-too-long"
+    # The line-too-long-disable is for
+    # the rather long URL that points to the original SciPy docs,
+    # and which cannot be broken up easily.
 
```

Comment:
I guess practicality beats purity? If better solutions are obvious to someone reading this, please feel invited to comment.

Reply:
It should be possible to insert a line break using a backslash like so:

```python
def func():
    """https://very.very/very/very/\
very/very/long/url

    Normal indentation.
    """
```

Note the missing indentation on the second line of the docstring

Reply:
Thanks!

Reply:
Sorry, I just noticed that this is an r-string. It will work if you make this a regular string literal, since the backslash escapes the line break in this case